src/cmd/initContainer.go

```diff
@@ -45,18 +57,22 @@ var (
 		user        string
 	}
 
-	initContainerMounts = []struct {
-		containerPath string
-		source        string
-		flags         string
-	}{
+	initContainerMounts = []initContainerMount{
 		{"/etc/machine-id", "/run/host/etc/machine-id", "ro"},
 		{"/run/libvirt", "/run/host/run/libvirt", ""},
 		{"/run/systemd/journal", "/run/host/run/systemd/journal", ""},
 		{"/var/lib/flatpak", "/run/host/var/lib/flatpak", "ro"},
 		{"/var/log/journal", "/run/host/var/log/journal", "ro"},
 		{"/var/mnt", "/run/host/var/mnt", "rslave"},
 	}
+
+	initContainerX11Symlinks = []initContainerSymlink{
```

Comment:
It would be better to do it as a bind mount instead. Or is there a reason why they must be symbolic links? Did I miss something?

For individual regular files we prefer symbolic links instead of bind mounts, because symbolic links can track updates to the original file. Whereas, with bound mounts, if the original file was updated atomically, which is what most programs do, then the atomic rename changes the inode, and the two diverge.

However, symbolic links are always weird and introduce various oddities (eg., stat versus lstat), so bind mount is better when possible.